### ISSUE TITLE
Folder to ISO: large folders gave corrupted image

### ISSUE THREAD

**Issue by gencer** [crystalidea/anytoiso#3]
I had a folder that contains 50.000-100.000 files in it. Not in one dir-only of course. Multiple nested dirs with many small and large files including application source codes to zipballs. If I do Folder to ISO, 17GB ISO Image gets created with UDF support. When I open it via any program such as WinRAR or UltraISO/PowerISO, I only see 3 folders (it has more actually) and even they are not complete either. PowerISO shows inside as 4GB but it's 17GB as on disk. Even, 7-zip can't open and says archive/file damaged.

It's not possible to give you those files due to nature and there is no debug/verbose log on your app as far as i know/see. How do we solve this problem? AnyToISO does this before and I lost important source codes. This time I checked output before I delete the source. I'm glad I checked.

**Comment by kleuter** [comment]
Can you make a smaller version of a folder (with files/subfolders) that reproduces the problem?

**Comment by gencer** [comment]
its only gets corrupted on large filsets. If file/folder counts are low, then it is OK.

**Comment by kleuter** [comment]
Can you please try the [beta version](https://www.crystalidea.com/downloads/anytoiso_setup_beta.exe)? It has more checks before creating an ISO image (file name length, directory path length).

**Comment by gencer** [comment]
First and only error I get is:

`09:21:49 - The '5b5a656e67615c4c617965725c5370616d42756e646c655c456e74697479...' filename is too long for the ISO image (max length - 127)`

But, I would like to ask that, I can understand that those files can be missing in ISO but even main directories are missing. For example;

Folder:
```
/
/test
/test/another/dir
/test/another/dir/....
/gencer
/kleuter
```

ISO Output:
```
/
/kleuter
```
even `/gencer` is dropped completely. But file size is correct. Content is corrupted.

**Comment by kleuter** [comment]
fixed in 3.9.0
